safe-global/safe-homepage
# fix: Ecosystem page issues

## What it solves

Part of #176 

- Scales down the background particle on mobile
- Also searches within the project descriptions and categories, adjusts the search input placeholder accordingly
- `sanitize` now leaves in target attribute if present

## How to test

1. Open `/ecosystem`
2. Search for a project description or category and see it in the results
3. Click on links at the bottom of the page and observe them opening in a new tab
